Add details about how to mitigate RavenDB high dirty memory issues (#7044)
Particular/docs.particular.net: servicecontrol/troubleshooting.md

When ServiceControl is restarted, the Corax-based index may get recreated. To prevent the ServiceControl instance from recreating the index, the index can be locked.

To lock an index, from the RavenDB studio, while ServiceControl is still in maintenance mode, look for the index that was set to use Lucene and click the `🔓 Unlocked` button. Change the setting to `🔒 Locked` ([Locked Ignore](https://ravendb.net/docs/article-page/7.0/csharp/client-api/operations/maintenance/indexes/set-index-lock#lock-modes)). The RavenDB studio will notify the operation completion with the message: _Lock mode was set to: Locked (ignore)_.

## RavenDB dirty memory

Each ServiceControl instance stores its data in a RavenDB database. RavenDB immediately writes data to the journal files and synchronizes writes to the data files in the background. The amount of data that needs to be flushed to disk is called "dirty memory."

Continuous dirty memory increases indicate too much pressure on the ServiceControl instance database. When that happens the following custom check message is presented:

> > There is a high level of RavenDB dirty memory (_dirty memory value in kb_). Check the ServiceControl troubleshooting guide for guidance on how to mitigate the issue. Visit the `https://docs.particular.net/servicecontrol/troubleshooting#ravendb-dirty-memory` page for more information.

Reviewer: Made some comments regarding the custom check text on the other PR that if accepted needs to be propagated to this section

Author: ```suggestion
> There is a high level of RavenDB dirty memory ({dirtyMemoryKb}kb). See `https://docs.particular.net/servicecontrol/troubleshooting#ravendb-dirty-memory` for guidance on how to mitigate the issue.
```
